netket/jax/_expect.py

```diff
@@ -95,6 +96,8 @@ def f(pars, σ, *cost_args):
 
     _, pb = nkvjp(f, pars, σ, *cost_args)
     grad_f = pb(dL̄)
+    grad_f = jax.tree_map(lambda x: mpi_sum_jax(x)[0], grad_f)
```

Comment:
porceddu baddu ohssignoreddu

Reply:
I understand you are not happy about this line?

Reply:
This is in accordance with the merged PR #1356: we need a `mpi_sum_jax` as explained in the docstring/example in `nk.jax.expect`.

Reply:
I mean, @alleSini99 , here you modified netket/jax/expect.py ?  I agree you should sum, but shouldn't you sum in the expect.py file of your operator?

Reply:
(I think this is a stray edit from a previous version originating before PR 1356)

Reply:
basically I think you should revert the change of this line?

Reply:
No, yes, you are right, let me fix it.